bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -405,6 +459,13 @@
             id="com.espressif.idf.ui.editor.heapTraceAnalysis"
             name="%editor.name.0">
       </editor>
+      <editor
+            class="com.espressif.idf.ui.installcomponents.InstallComponentsEditor"
+            default="false"
+            icon="icons/ESP-IDF Install Tools.png"
```

Comment:
Please request for a new icon and replace it here

Reply:
Done